.github/dependabot.yml

```diff
@@ -1,9 +1,12 @@
 version: 2
 updates:
-- package-ecosystem: npm
-  directory: "/"
-  schedule:
-    interval: weekly
-    time: "10:00"
-  open-pull-requests-limit: 10
-  versioning-strategy: increase
+  - package-ecosystem: npm
+    directory: '/'
+    schedule:
+      interval: weekly
+      time: '10:00'
+    open-pull-requests-limit: 10
+    versioning-strategy: increase
+    ignore:
+      - dependency-name: '*'
```

Comment:
Do we want to allow this for indirect deps as well or just focus on the 'direct' dependencies?

As using the `*` will mean it does all and just results in the lockfile updating and that change then not making it to the consumer